python/tests/conftest.py

```diff
@@ -249,7 +248,9 @@ def version_store_factory(arcticdb_test_lmdb_config, lib_name):
     `name` can be a magical value "_unique_" which will create libs with unique names."""
     used = {}
 
-    def version_store(col_per_group: Optional[int] = None, row_per_segment: Optional[int] = None, **kwargs) -> NativeVersionStore:
+    def version_store(
+        col_per_group: Optional[int] = None, row_per_segment: Optional[int] = None, **kwargs
+    ) -> NativeVersionStore:
```

Comment:
Please avoid checking in random formatting/linting changes like this in files that are otherwise untouched